src/main/java/se/bjurr/violations/lib/reports/Reporter.java

```diff
@@ -45,6 +45,7 @@
       Parser.GENDARME,
       "http://www.mono-project.com/docs/tools+libraries/tools/gendarme/",
       ""),
+  GENERIC("Generic reporter", Parser.FILEPARSER, "", ""),
   GOLINT("GoLint", Parser.GOLINT, "https://github.com/golang/lint", ""),
```

Comment:
I think the parser should also be named GENERIC. The parsed conten may not come from a file.

Also some short description is needed.

Perhaps: `GENERIC("Generic reporter", Parser.GENERIC, "", "Will create one single violation with all the content as message."),`

Reply:
Made suggested changes.